### ISSUE TITLE
NVDA doesn't read selected content in microsoft access

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2281]
**Reported by ashleycox on 2012-04-27 10:35**
When a database table is open in Microsoft access, and shift+right/left/up/down arrows or control+space are used to select columns, NVDA simply announces "grid" and doesn't announce the selected content. Ideally NVDA would read this selection data much like it does in excel.

**Comment by bhavyashah** [comment]
On the basis of https://github.com/nvaccess/nvda/issues/2281#issuecomment-155297372, could the NeedsCodeReview label please be added to this ticket? @josephsl @ehollig

**Comment by Adriani90** [comment]
@josephsl I know this is a quite old one, but do you still have the code for fixes available? Could you raise a PR for this? Or is it nor relevant anymore?

**Comment by Brian1Gaff** [comment]
I'd like it for  an old xp machine. Its a real pain at times. Why Microsoft 
decided to use a completely different input output system in Access tan 
anything else eludes me. The original UIA solution did not work well as on 
XP it only worked with an optional update, but tended to lock up for long 
periods due to bugs.
 I do remember this from a long time ago...
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
-----

**Comment by Adriani90** [comment]
No meaningful updates during the last 7 years, closing as abandoned.
